### PR TITLE
Add WP Admin Dashboard to Home Quick Links if Nav Unification is enabled

### LINKED ISSUE #50945: Nav Unification: Make calypso "My Home" always available
### What
I noticed that the calypso “My Home” page is changed to the WP Admin “home dashboard" when you make the changes in "Account settings" to always show the WP Admin pages when possible:

markup-2021-03-09-at-16 52 34-1

**This is expected as per the scope of the project and what this button should change in calypso**.

In summary:
- with “always show the WP Admin pages when possible” toggle ON;
- then "My Home" is showing the WP Admin dashboard instead of Calypso My Home;
- so you have no UI to go to https://wordpress.com/home/ anymore, you can only access https://[SITE_DOMAIN]/wp-admin/index.php in this context.

I have a gut feeling though that maybe we shouldn't do that for "My Home", so my suggestion is to keep the calypso "My Home" enabled as opposed to hiding it when this toggle is enabled. Alternatively, we could add “WP Dashboard” as a separate link directly under “My Home” when the “Always show WP Admin pages” account setting toggle is turned on.

Pinging @sfougnier here for thoughts.

### Why
We do so much important stuff there in the calypso "My Home" from Marketing, Learning, Support, and other focuses we have at any given point in time in WP.com, we shouldn't hide calypso My Home from users - it may also make it challenging for AB tests and campaigns moving forward.

**Additional context**
Initially discussed in Slack, here: p1615375900105500-slack-C01APA37DDW

### PR DESCRIPTION
**Pull request by tjcafferkey**
#### Changes proposed in this Pull Request

* Add WP Admin Dashboard to quick links on Home screen
* Only add if Nav Unification is enabled because there is an existing link in the old navigation

#### Testing instructions

* Go to the Home screen on a site of your choice
* Find WP Admin Dashboard as a link under the "Quick Links" section.

**Before**
Screenshot 2021-07-05 at 09 21 00

**After**
Screenshot 2021-07-05 at 09 20 48

Fixes https://github.com/Automattic/wp-calypso/issues/50945

### CODE REVIEW

File: client/my-sites/customer-home/cards/actions/quick-links/index.jsx [hunk 1]
```diff
@@ -138,6 +142,14 @@ export const QuickLinks = ( {
 					gridicon="domains"
 				/>
 			) }
+			{ isUnifiedNavEnabled && siteAdminUrl && (
+				<ActionBox
+					href={ siteAdminUrl }
+					hideLinkIndicator
+					gridicon="my-sites"
+					label={ translate( 'WP Admin Dashboard' ) }
```
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/actions/quick-links/index.jsx:150]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **2** times:
`translate( 'wp-admin Dashboard' )` *ES Score: 8*

**Comment by BogdanUngureanu** [comment]
Just tested it and works great! 🎉  🚀

**Review by BogdanUngureanu — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6145375

Thank you @tjcafferkey for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6145375).

**Pull request merged**